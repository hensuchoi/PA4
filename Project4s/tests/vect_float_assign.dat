funct: vectfloatassign
gin: f, float, 0.25
gin: v2, vec2, 2.1, 3.2
