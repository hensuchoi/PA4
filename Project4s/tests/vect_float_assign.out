Result: 6.720000e+00
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
segfault vect_float_assign!
